docs/source/use_cases/face_mask_detection.rst

```diff
@@ -58,7 +58,7 @@ These are the nodes used in the earlier demo (also in |pipeline_config|_):
 .. code-block:: yaml
 
    nodes:
-   - input.live
+   - input.visual
```

Comment:
Demo intended to be run using webcam to interact with use case better.
```suggestion
   - input.visual:
       source: 0
```
Same for group size checking, object_counting_present, privacy_protection_faces, social_distancing, zone_counting.